proposals/description/tip_vebalotc.ts

```diff
@@ -0,0 +1,94 @@
+import { ethers } from 'ethers';
+import { TemplatedProposalDescription } from '@custom-types/types';
+
+const tip_vebalotc: TemplatedProposalDescription = {
+  title: 'TIP-121c: veBAL OTC',
+  commands: [
+    // 1. Handle veBAL OTC
+    {
+      target: 'proxyAdmin',
+      values: '0',
+      method: 'changeProxyAdmin(address,address)',
+      arguments: (addresses) => [addresses.balancerGaugeStaker, addresses.aaveCompaniesMultisig],
+      description: 'Transfer proxy ownership of balancerGaugeStaker to aaveCompaniesMultisig'
+    },
```

Comment:
Todo: upgrade implementation before proposal execution, to allow boost delegation